client/galaxy/scripts/onload.js

```diff
@@ -19,6 +19,7 @@ var POPUPMENU = require( 'ui/popupmenu' );
 window.make_popupmenu = POPUPMENU.make_popupmenu;
 window.make_popup_menus = POPUPMENU.make_popup_menus;
 window.init_tag_click_function = require( 'ui/autocom_tagging' );
+window.Toastr = require( 'libs/toastr' );
```

Comment:
I totally agree, but can I use `require()` inside of a script in the `<head></head>` section?